### ISSUE TITLE
Loading from checkpoint

### ISSUE THREAD

**Issue by pszmk** [D4L-Pigeons/D4L-Hackaton#42]
Downloading checkpoint from neptune into .neptune and loading it in main script. Code must be aware of the past epochs e.g. loss coef schedulers should use the info about current epoch and the state_dict should be properly loaded.

Done but must be tested.

**Comment by pszmk** [comment]
should neptune logger "continue the same plots" - i guess no, but the number of epochs should be correct in the logs of the run started from checkpoint
